checkov/github_actions/runner.py

```diff
@@ -69,18 +70,31 @@ def included_paths(self) -> Iterable[str]:
         return [".github"]
 
     def get_resource(self, file_path: str, key: str, supported_entities: Iterable[str],
-                     definitions: dict[str, Any] | None = None) -> str:
-        if not definitions:
-            return key
-
-        potential_job_name = key.split('.')[1]
-        if potential_job_name != '*':
-            new_key = f'jobs.{potential_job_name}'
-        else:
-            start_line, end_line = self.get_start_and_end_lines(key)
-            job_name = Runner.resolve_job_name(definitions, start_line, end_line)
-            step_name = Runner.resolve_step_name(definitions["jobs"][job_name], start_line, end_line)
-            new_key = f'jobs.{job_name}.steps.{step_name}'
+                     start_line: int = -1, end_line: int = -1) -> str:
+        """
+        supported resources for GHA:
+            jobs
+            jobs.<job_name>.steps<#step_number>[<step_name>]
```

Comment:
I think we should align our resource names format throughout the runners
Why not follow the current method of
`jobs(job_name).steps[step_index](step_name)`?
@gruebel following our conversation

Reply:
yeah you are right

Reply:
actually, that should be the entity, and I fixed it to `jobs.*.steps[]`.
But I changed all the names of circleCI and GHA for this convention. 🍒